Makefile

```diff
@@ -0,0 +1,7 @@
+protos: go-proto
+
+go-proto:
+	docker rmi -f gtfs-golang
+	docker build -t gtfs-golang -f golang/Dockerfile . && docker run -it gtfs-golang && docker run --name gtfs-golang-container -it -d gtfs-golang
```

Comment:
Why are these three commands on the same line?

Reply:
Ah, not relevant anymore since I removed the `Makefile`, but they should be broken up! I did so in the new shell script